jib-core/src/main/java/com/google/cloud/tools/jib/builder/steps/WriteTarFileStep.java

```diff
@@ -58,7 +58,9 @@ public BuildResult call() throws ExecutionException, InterruptedException, IOExc
     try (ProgressEventDispatcher ignored =
         progressEventDispatcherFactory.create("writing to tar file", 1)) {
       // Builds the image to a tarball.
-      Files.createDirectories(outputPath.getParent());
+      if (outputPath.getParent() != null) {
+        Files.createDirectories(outputPath.getParent());
```

Comment:
Does this work correctly when the working directory is not the project root? For example, `mvn -f examples/helloworld/pom.xml compile ...`.

Reply:
Yup